istioctl/pkg/install/pre-check.go

```diff
@@ -0,0 +1,159 @@
+package install
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+
+	authorizationapi "k8s.io/api/authorization/v1beta1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/version"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
+)
+
+const (
+	minK8SVersion = "1.11"
+)
+
+type targetResource struct {
+	namespace string
+	group     string
+	version   string
+	name      string
+}
+
+func installPreCheck(istioNamespaceFlag *string, restClientGetter resource.RESTClientGetter, writer io.Writer) error {
+	fmt.Fprintf(writer, "\n")
+	fmt.Fprintf(writer, "Checking the cluster to make sure it is ready for Istio installation...\n")
+	fmt.Fprintf(writer, "\n")
+	fmt.Fprintf(writer, "kubernetes-api\n")
```

Comment:
"Kubernetes" should be capitalized where it appears to the user (except as part of a code fragment such as a package name).

Reply:
done